integration_tests/models/schema_tests/schema.yml

```diff
@@ -32,11 +32,21 @@ models:
     tests:
       - dbt_utils.equal_rowcount:
           compare_model: ref('test_equal_rowcount')
-      
+
   - name: data_people
     columns:
       - name: is_active
         tests:
           - dbt_utils.cardinality_equality:
               field: is_active
-              to: ref('data_people')
\ No newline at end of file
+              to: ref('data_people')
+
+  - name: data_test_relationships_where_table_2
+    columns:
+      - name: id
+        tests:
+          - dbt_utils.relationships_where:
+              from: id
+              to: ref('data_test_relationships_where_table_1')
+              field: id
+              from_condition: id <> '4'
```

Comment:
I think this test is failing on BigQuery because the `id` column is being loaded as an integer. Can you change `'4'` to `4` here? I think we should be good to go after that!

Reply:
Oops. Bad habit from Snowflake where `4 = '4'`... 
Fixed!